### PR TITLE
5651: update toggle svgs, adding title option and styles

### PR DESCRIPTION
**Pull request by transitorytammy**

### CODE REVIEW

**Comment by lawrencegetfeedback** [comment]
Just documenting our offline convo:
- We probably want to decide on whether we use the `title` HTML attribute, since it has the side effect of browsers rendering tooltips. I think we want to use bourbon tooltips across the board for visual tooltips. Looping in @alexhking .
- 🍹 The svg assets can probably be deduped, and just set an opacity for when the component is `disabled`.
- Good work cleaning up the code. 👍

File: addon/components/bourbon-toggle.js [hunk 1]
```diff
@@ -1,58 +1,40 @@
...
   ariaRole: 'button',
-
   value: null,
-  toggleState: null,
   disabled: false,
   action: null,
-  readOnly: false,
-  label: 'toggle button',
+
+  label: computed('title', 'value', 'disabled', function() {
```
Comment (lawrencegetfeedback) [inline addon/components/bourbon-toggle.js]: 🍹 Do we need to preserve the `toggle button` in the aria label? Just wondering what we want as the end result to be read out by screen-readers.

File: stories/Toggle/toggle.md [hunk 2]
```diff
@@ -11,6 +11,7 @@
 | propertyOptions | type | defaultValue | isRequired | description | options |
 |----------|:----------|:--------------|:------------|:-------------|:------|
 | value | boolean | null | true | determines if toggle is on or off ||
+| title | string | null | false | pass in title to explain state of toggle | if nothing is passed then a generic title will be shown on hover|
```
Comment (lawrencegetfeedback) [inline stories/Toggle/toggle.md]: I guess this line isn't needed, since we're not using `title` anymore.

File: stories/Toggle/toggle.md [hunk 3]
```diff
@@ -11,6 +11,7 @@
...
 | toggleState | string | null | true | computed on init and determines the SVG to be displayed ||
```
Comment (lawrencegetfeedback) [inline stories/Toggle/toggle.md]: Not computed on init anymore, should probably update this line.

File: stories/Toggle/toggle.md [hunk 4]
```diff
@@ -11,7 +11,7 @@
 | propertyOptions | type | defaultValue | isRequired | description | options |
 |----------|:----------|:--------------|:------------|:-------------|:------|
 | value | boolean | null | true | determines if toggle is on or off ||
+| label | string | computed | tru | a computed property to provide aria-label describing state of toggle | |
```
Comment (lawrencegetfeedback) [inline stories/Toggle/toggle.md]: typo: `tru`

**Comment by transitorytammy** [comment]
.25 day in march

**Pull request merged**